packages/cashier-v2/src/lib/Transfer/components/TransferForm/TransferForm.tsx

```diff
@@ -20,6 +26,20 @@ const TransferForm = () => {
         toAmount: '',
     };
 
+    const getDailyTransferCountLimit = (
+        fromAccount: TTransferFormikContext['fromAccount'],
+        toAccount: TTransferFormikContext['toAccount']
+    ) => {
+        if (accountLimits?.daily_transfers) {
+            if (fromAccount?.account_type && fromAccount?.account_type !== 'binary') {
+                return (accountLimits?.daily_transfers[fromAccount?.account_type] as TAccountLimits).available;
+            } else if (toAccount?.account_type && toAccount?.account_type !== 'binary') {
+                return (accountLimits?.daily_transfers[toAccount?.account_type] as TAccountLimits).available;
+            }
+            return (accountLimits?.daily_transfers.internal as TAccountLimits)?.available;
+        }
+    };
+
```

Comment:
Hi team, Due to the broken types received from `@deriv/api-types` we need to explicitly type-cast on line 35, 37 and 39.
It would be great if we can find a solution for this.
cc: @nijil-deriv @wojciech-deriv @heorhi-deriv 

![Screenshot 2024-04-26 at 10 02 18 AM](https://github.com/binary-com/deriv-app/assets/125039206/835b0e9f-d09c-4e93-8927-723d0e77a0a9)